src/library/scala/math/Numeric.scala

```diff
@@ -80,6 +81,8 @@ object Numeric {
     def toLong(x: Short): Long = x.toLong
     def toFloat(x: Short): Float = x.toFloat
     def toDouble(x: Short): Double = x.toDouble
+    // delegate to math.signum(Int)
```

Comment:
IMHO, this comment does not add anything to what can be read from the code.

Reply:
At the time it felt somewhat notable that it's using `Int`'s facility, but you're right that adding this comment doesn't add much.

Reply:
Addressed in 430c7acb33d41ea95360f613fe9794387fd488b8